tests/packages/utils/test_utils.py

```diff
@@ -41,7 +41,10 @@
                 ' "win32" and python_version < "3.6" and python_version >= "3.3" or'
                 ' sys_platform == "win32" and python_version < "3.3"'
             ),
-            {"python_version": [[("<", "3.6")]], "sys_platform": [[("==", "win32")]]},
+            {
+                "python_version": [[("<", "3.6")], [("<", "3.3")]],
+                "sys_platform": [[("==", "win32")]],
+            },
```

Comment:
That looks like a regression, doesn't it?

Reply:
I don't think it's a problem - per some of the other test cases here, `convert_markers` isn't too precise anyway.

Actually what this testcase reveals is a change in behaviour at `parse_marker()` which gets _some_ of the available simplifications by using the `.of()` methods - but misses out on the simplifications achieved by `cnf()` and `dnf()` in `union()` and `intersect()`.

I have another simplification in my pocket that reworks `_compact_markers()` in terms of `intersect()` and `union()`.   That results in parsed markers being as simplified as we can manage.

It's a little more disruptive to the unit test but I think I'm good with it, will push as a separate commit shortly - see what you think.